src/assets/accessories/bondage_belt/bondage_belt.asset.ts

```diff
@@ -0,0 +1,59 @@
+DefineAsset({
+	name: 'Bondage Belt',
+	size: 'medium',
+	graphics: 'graphics.json',
+	colorization: {
+		cuff: {
+			name: 'Cuff',
+			default: '#222222',
+		},
+		belt: {
+			name: 'Belt',
+			default: '#000000',
+		},
+		smallRings: {
+			name: 'Eyelets',
+			default: '#FFFFFF',
+		},
+		largeRings: {
+			name: 'D-Rings',
+			default: '#FFFFFF',
+		},
+	},
+	// size:250, y:500, centered
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Restraint',
+			'Restraint_arms',
+			'Belt_chainable',
+		],
+	},
+	modules: {
+		lock: {
+			type: 'lockSlot',
+			name: 'Lock for belt',
+			occupiedProperties: {
+				blockAddRemove: true,
+			},
+		},
+	},
+	chat: {
+		actionAdd: 'SOURCE_CHARACTER fastened a bondage belt around TARGET_CHARACTER_DYNAMIC_POSSESSIVE taille.',
+		actionRemove: 'SOURCE_CHARACTER loosened and slipped off the bondage belt from TARGET_CHARACTER_DYNAMIC_POSSESSIVE taille.',
+	},
```

Comment:
"Taille" is a German word and I do not think it exists in English or is too uncommon to understand for a native speaker. Waist is the common word  here.